.editorconfig

```diff
@@ -37,7 +37,7 @@ charset = utf-8
 indent_size = 4
 # isort plugin configuration
 known_first_party = invenio_indexer
-known_third_party = invenio_db,invenio_search,invenio_records
+known_third_party = invenio_db,invenio_search,invenio_records,invenio_marc21
```

Comment:
Invenio-Indexer has no dependency on Invenio-MARC21 - i.e. it shouldn't be listed here.